webapp/tests/mocha/unit/translator.spec.js

```diff
@@ -0,0 +1,65 @@
+const BootstrapTranslator = require('../../../src/js/bootstrapper/translator'),
+      { expect } = require('chai');
+
+const translationData = {
+  en: {
+    LOAD_ASSETS:   'Loading assets!',
+    FETCH_INFO:    x => `Fetching info (${x} docs)!`,
+    EN_ONLY:       'Yay!',
+  },
+  es: {
+    LOAD_ASSETS:   'Cargando activos!',
+    FETCH_INFO:    x => `Descarga de datos (${x} documentos)!`,
+    ES_ONLY:      'Vamos!',
```

Comment:
This value is unused.  Is it useful to keep it here?

Reply:
No